packages/@eventual/aws-cdk/src/command-service.ts

```diff
@@ -401,23 +403,22 @@ export class CommandService<Service = any> {
     // Allow them to access any of the methods on the service client by default.
     this.props.service.configureForServiceClient(handler);
     this.grantInvokeHttpServiceApi(handler);
-    /**
-     * Entity operations
-     */
+    // Entity operations
     this.props.entityService.configureReadWriteEntityTable(handler);
     this.props.entityService.configureInvokeTransactions(
       this.systemCommandsHandler
     );
-    /**
-     * Bucket Operations
-     */
+    // Bucket Operations
     this.props.bucketService.configureReadWriteBuckets(handler);
+    // Search operations
+    this.props.searchService?.configureSearch(handler);
   }
 
   private configureSystemCommandHandler() {
     // for update task
     this.props.taskService.configureWriteTasks(this.systemCommandsHandler);
     this.props.taskService.configureCompleteTask(this.systemCommandsHandler);
+    this.props.searchService?.configureSearch(this.systemCommandsHandler);
```

Comment:
Is this needed? When would a system command call the index?

Reply:
Good question. Let me check. I was kind of spraying and praying while debugging

Reply:
The system commands have a separate set of dependencies, I don't think you added search to it.

Reply:
Removing this and will see if tests pass. Code suggests I don't need it.